app/controllers/api/generic_works_controller.rb

```diff
@@ -0,0 +1,98 @@
+module API
+  # Adds an endpoint that customizes the update action on nested works
+  class GenericWorksController < ApplicationController
```

Comment:
Oh! I see! I was looking for the "Resource Controller" and couldn't find it. So I was confused what you were talking about. We actually thought about putting it there. We are looking into the patch fix for the routes, and if that works, it seems as though we can remove this module all together. (from what I understand :P)